packages/react-checkbox/Spec.md

```diff
@@ -2,62 +2,267 @@
 
 ## Background
 
-_Description and use cases of this component_
+Checkboxes are used to represent one or more options, giving the user a way to select one or more of these options.
 
 ## Prior Art
 
-_Include background research done for this component_
+- Open UI research:
+  - Concepts: https://open-ui.org/components/checkbox.research.concepts
+  - Findings: https://open-ui.org/components/checkbox.research.findings
+  - Proposal: https://open-ui.org/components/checkbox
+- Github Epic: https://github.com/microsoft/fluentui/issues/18454
 
-- _Link to Open UI research_
-- _Link to comparison of v7 and v0_
-- _Link to GitHub epic issue for the converged component_
+### [Checkbox in v0/Northstar](https://fluentsite.z22.web.core.windows.net/0.56.0/components/checkbox/definition)
+
+The v0 `Checkbox` component does not support a `indeterminate` state which is needed for the converged checkbox. A `div` with `role="checkbox"` and does not use the native input element.
```

Comment:
nit:

```suggestion
The v0 `Checkbox` component does not support an `indeterminate` state which is needed for the converged checkbox. It is rendered as a `div` with `role="checkbox"` and does not use the native input element.
```

Reply:
![image](https://user-images.githubusercontent.com/20744592/123265059-f0b15c80-d4fa-11eb-8d46-255040135c84.png)

The terminology in v0 is different, we call it `mixed`, it follows from the `aria-checked="mixed"` value

Reply:
Yes, vNext follows v8's terminology (indeterminate) based on the design spec.

Reply:
Yes but change the statement 

> The v0 `Checkbox` component does not support a `indeterminate` state which is needed for the converged checkbox

mixed/indeterminate are the same thing in v0/v8